vroom/clangformat.vroom

```diff
@@ -181,3 +181,14 @@ current buffer.
 
 The -assume-filename arg passed above is also related, used by the clang-format
 tool to detect any custom style rules particular to the current file.
+
+
+Bug #102. When the buffer is empty, clang-format does _not_ return a cursor
+position
+
+  @clear
+  @messages (STRICT)
+  :call maktaba#error#TryCommand('FormatCode clang-format')
```

Comment:
Does it actually throw errors in the error case, or just warn? It might actually work fine if you just directly invoke `:FormatCode clang-format`, but fine to keep it this way if there is a reason that gets weird.

Reply:
Yup, this is simply a carryover from me desperately trying to write a unit test that actually did what I wanted :-P

I've updated the test as you suggested.